src/server/game/AI/SmartScripts/SmartScriptMgr.cpp

```diff
@@ -340,7 +340,6 @@ void SmartAIMgr::LoadSmartAIFromDB()
             }
             break;
         case SMART_EVENT_VICTIM_CASTING:
-        case SMART_EVENT_IS_BEHIND_TARGET:
```

Comment:
Is this change related to the PR ?

Reply:
Sneaky change for an already deprecated event

Reply:
that should be mentioned under **Changes proposed:**